rasa_sdk/forms.py

```diff
@@ -689,3 +690,61 @@ def _get_entity_type_of_slot_to_fill(
                 return None
 
         return entity_type
+
+
+class FormValidationAction(Action, ABC):
+    """An action that validates if every extracted slot is valid."""
+
+    async def validate(
+        self,
+        dispatcher: "CollectingDispatcher",
+        tracker: "Tracker",
+        domain: "DomainDict",
+    ) -> List[EventType]:
+        """Validate slots by calling a validation function for each slot.
+
+        Args:
+            dispatcher: the dispatcher which is used to
+                send messages back to the user.
+            tracker: the conversation tracker for the current user.
+            domain: the bot's domain.
+        Returns:
+            `SlotSet` events for every validated slot.
+        """
+        slots: Dict[Text, Any] = tracker.form_slots_to_validate()
```

Comment:
I renamed it this morning (tagged you there)
```suggestion
        slots: Dict[Text, Any] = tracker.slots_to_validate()
```